docs/partials/cli-install.md

```diff
@@ -11,19 +11,28 @@ scoop install infra
 ```
 {% /tab %}
 
-{% tab label="Linux" %}
-
-#### Ubuntu & Debian
+{% tab label="Ubuntu & Debian" %}
+Download the [latest][1] Debian package from GitHub and install it with `dpkg` or `apt`.
 ```
-echo 'deb [trusted=yes] https://apt.fury.io/infrahq/ /' | sudo tee /etc/apt/sources.list.d/infrahq.list
-sudo apt update
-sudo apt install infra
+sudo dpkg -i infra_*.deb
 ```
-
-#### Fedora & Red Hat Enterprise Linux
 ```
-sudo dnf config-manager --add-repo https://yum.fury.io/infrahq/
-sudo dnf install infra
+sudo apt install ./infra_*.deb
+```
+{% /tab %}
+{% tab label="Fedora & Red Hat Enterprise Linux" %}
+Download the [latest][1] RPM package from GitHub and install it with `rpm` or `dnf`.
+```
+sudo rpm -i infra-*.rpm
+```
+```
+sudo dnf install infra-*.rpm
 ```
 {% /tab %}
 {% /tabs %}
+
+## Manual Install
+
+Download the [latest][1] release from GitHub, unpack the file, and add the binary to the `PATH`.
```

Comment:
Would make this a hyperlink for now since I don't think footnotes will render well in the docs (yet)

Reply:
This is a hyperlink, just a different style. Footnotes would be `[^1]` if I recall correctly. You can check by going to the preview and clicking `latest`

Reply:
TIL! That's awesome. Looks good then